phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixLoggingMetricsIT.java

```diff
@@ -157,7 +157,7 @@ public void testPhoenixMetricsLoggedOnClose() throws Exception {
         // Hence mutation metrics are not expected during connection close
         loggedConn.close();
         assertTrue("Mutation write metrics are not logged for " + tableName2,
-                mutationWriteMetricsMap.size() == 0);
+                mutationWriteMetricsMap.size() > 0);
```

Comment:
the text for `assertTrue`, and the comment above, should be updated to reflect the new reality.